db.go

```diff
@@ -237,6 +325,11 @@ func (db *DB) Name() string {
 	return db.name
 }
 
+// SecondaryPath returns the secondary path of the database, if it is a secondary database instance.
+func (db *DB) SecondaryPath() string {
+	return db.secondaryPath
+}
```

Comment:
What is the purpose of this method? Don't the callers know what they are passing to `OpenDbAsSecondary`?

Reply:
It was to keep symmetry with `name`/`Name()`, which keeps track of the primary path